ts/api/pagopa.ts

```diff
@@ -134,12 +134,80 @@ type GetWalletsUsingGETExtraT = MapResponseType<
   WalletListResponse
 >;
 
+/**
+ *
+ * sanitaze psp tags if it is an array with values (avoid no string value and duplicates)
+ * otheriwise set tags as an empty array
+ * @param w the wallet object
+ */
+const fixWalletPspTagsValues = (w: any) => {
```

Comment:
use `unknown` instead of `any`

```suggestion
const fixWalletPspTagsValues = (w: unknown) => {
```

Reply:
@cloudify If I use **unknown** type then I can't access **w** properties. I need to know the exact type of **w** and in this case it is an object literal (coming from json payload)

Reply:
that's the point, you don't know, you need a decoder (as you did below)